test/util/cart_utils.py

```diff
@@ -301,3 +311,27 @@ def print(self, cmd):
 
         self.stdout.info(cmd)
         self.progress_log.info(cmd)
+
+    def log_check(self, cartobj):
+	"""Check log files for consistency """
+
+	logparse = cartobj.params.get("logparse", "/run/tests/*/")
+        if logparse is None:
+		return
+
+        """Check log files for consistency """
+        strict_test = False
+	print ("Parsing log path", cartobj.log_path)
+	if not os.path.exists(cartobj.log_path):
+		print ("Path does not exist")
+		return
+
+	for filename in os.listdir(cartobj.log_path):
```

Comment:
I am not sure I understand. Are you saying to sort the files passed upon the pid appended and parse them in that order. What would the advantage of doin that be if that is the case?

Reply:
I was thinking that the suffix would be a rank ID rather than a pid.  The advantage would be if the same errors were generated by the same ranks then they'd get reported in the same order.

Reply:
Going forward if you have multiple clients, neither of clients (in non-pmix mode) will have a rank (rank set to NO_RANK for those). As such we cant really use rank to differentiate logs between multiple clients

Reply:
So I guess we leave this one as is?

Reply:
Yes, leave as is.